tests/composer/autocomplete_dropdown_component.test.ts

```diff
@@ -113,6 +113,15 @@ describe("Functions autocomplete", () => {
       expect(fixture.querySelectorAll(".o-autocomplete-value")[1].textContent).toBe("SZZ");
     });
 
+    test("Autocomplete dropdown text should be black by default", async () => {
```

Comment:
doesn't really test the original bug:

- the problem was only on data validation drop down, not on function proposal
- we don't check if the cell style affects the proposals anymore